contracts/test/strategies/morpho-aave.mainnet.fork-test.js

```diff
@@ -10,7 +10,7 @@ const {
 const { createFixtureLoader, morphoAaveFixture } = require("../_fixture");
 const { impersonateAndFund } = require("../../utils/signers");
 
-describe("ForkTest: Morpho Aave Strategy", function () {
+describe.skip("ForkTest: Morpho Aave Strategy", function () {
```

Comment:
Why don't we delete the whole file instead? To keep it in case we plan to build another test like this?

Reply:
Indeed, it make sense!